app/views/teaching_events/about_git_events.html.erb

```diff
@@ -0,0 +1,101 @@
+<%= render Campaign::HeaderComponent.new(
+  title: ["Get Into Teaching", "events"],
+  colour: "yellow",
+  image: "media/images/content/hero-images/0025.jpg"
```

Comment:
Are we changing this image @gazcarr or leaving as is?

Reply:
I think this image works fine here